products/digital-sales-rooms/installation.md

```diff
@@ -0,0 +1,181 @@
+---
+nav:
+   title: Digital Sales Rooms installation
+   position: 10
+
+---
+
+# Digital Sales Rooms Installation
+
+::: warning
+To access the *Digital Sales Rooms* source code, please create a support ticket in your Shopware Account and share your GitLab (not GitHub) username. You will then be granted access to the private repository.
+:::
+
+## Installation and usage
+
+To install and use the *Digital Sales Rooms* feature, follow the below steps:
+
+### Get the plugin
+
+::: warning
+The term "Digital Sales Rooms" has been rebranded as "Digital Sales Rooms", though no modifications have been implemented in the source code at this point.
```

Comment:
@rittou is this meant? There is no difference in the rebrand

> The term "Guided shopping" has been rebranded as "Digital Sales Rooms"

Reply:
Yeah, my bad.
It should be `Guided Shopping` -> `Digital Sales Rooms`, and there are many modifications regarding the name change.